### PR TITLE
signup_signin

### PR DESCRIPTION
**Pull request by Lyst67**
Є реестрація і логін. Поки що без валідації. Стилі виправив.

### CODE REVIEW

File: src/pages/SignInPage/SignInPage.jsx [hunk 1]
```diff
@@ -1,5 +1,74 @@
...
+                <span>
+                  <Link className={css.signup_link} to="/signup">
+                    Sign Up
+                  </Link>
+                </span>
+              </p>
+            </div>
+          </div>
+
+          <div className={css.welcome_bgd}>
```
Comment (Sokilskill) [inline src/pages/SignInPage/SignInPage.jsx:40]: це винести вкомпонент

File: src/pages/WelcomePage/WelcomePage.jsx [hunk 2]
```diff
@@ -1,52 +1,77 @@
...
-                    <p className={css.views}>350 +</p>
-                    <p className={css.views_text}>Video tutorial</p></div>    
-                    </div>      
-          <div className={css.cal_box}>
-            <div className={css.cal_ellips}>
-            <svg className={css.icon_run}>
-                 <use className={css.icon_run} href={`${sprite}#icon-running-stick-figure-svgrepo-com-1-1`} />
-               </svg>
+          </div>
+          <div className={css.welcome_bgd}>
```
Comment (Sokilskill) [inline src/pages/WelcomePage/WelcomePage.jsx:44]: винести в компонент

**Review by Sokilskill — APPROVED** [review]
винеси в окремий компонент елементи

**Comment by Lyst67** [comment]
Ок. Зроблю.

15 січня 2024, 09:26:49, від "Serhii" ***@***.***>:


@Sokilskill approved this pull request.
винеси в окремий компонент елементи
In src/pages/SignInPage/SignInPage.jsx:
> +                  platform:
+                </p>
+              </div>
+              <SignInForm onSubmit={handleFormData} />
+              <p className={css.signup_option}>
+                Don’t have an account?
+                <span>
+                  <Link className={css.signup_link} to="/signup">
+                    Sign Up
+                  </Link>
+                </span>
+              </p>
+            </div>
+          </div>
+
+          <div className={css.welcome_bgd}>
це винести вкомпонент
In src/pages/WelcomePage/WelcomePage.jsx:
>              </div>
-                    <div>
-                    <p className={css.views}>350 +</p>
-                    <p className={css.views_text}>Video tutorial</p></div>    
-                    </div>      
-          <div className={css.cal_box}>
-            <div className={css.cal_ellips}>
-            <svg className={css.icon_run}>
-                 <use className={css.icon_run} href={`${sprite}#icon-running-stick-figure-svgrepo-com-1-1`} />
-               </svg>
+          </div>
+          <div className={css.welcome_bgd}>
винести в компонент
—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you authored the thread.Message ID: ***@***.***>

--=-wVOo+DNbh/Yi1KH2DSm0
Content-Type: text/html; charset="UTF-8"
Content-Transfer-Encoding: base64

PGh0bWw+PGJvZHk+PHNwYW4gc3R5bGU9ImRpc3BsYXk6YmxvY2s7IiBjbGFzcz0ieGZtXzQ3MjEy
NTIzIj48ZGl2PjxzcGFuIHN0eWxlPSJmb250LXNpemU6MTJwdDtsaW5lLWhlaWdodDoxNHB0O2Zv
bnQtZmFtaWx5OkFyaWFsOyIgY2xhc3M9InhmbWMxIj7QntC6LiDQl9GA0L7QsdC70Y4uPC9zcGFu
PjwvZGl2Pg0KPGRpdj48YnIvPjwvZGl2Pg0KPGRpdj48aT48c3BhbiBzdHlsZT0iZm9udC1zaXpl
OjEwcHQ7bGluZS1oZWlnaHQ6MTJwdDsiPjxzcGFuIHN0eWxlPSJmb250LWZhbWlseTpBcmlhbDsi
PjE1INGB0ZbRh9C90Y8gMjAyNCwgMDk6MjY6NDksINCy0ZbQtA0KICAgICJTZXJoaWkiICZsdDs8
L3NwYW4+PGEgaHJlZj0ibWFpbHRvOm5vdGlmaWNhdGlvbnNAZ2l0aHViLmNvbSIgdGFyZ2V0PSJf
YmxhbmsiPjxzcGFuIHN0eWxlPSJmb250LWZhbWlseTpBcmlhbDsiPm5vdGlmaWNhdGlvbnNAZ2l0
aHViLmNvbTwvc3Bhbj48L2E+PHNwYW4gc3R5bGU9ImZvbnQtZmFtaWx5OkFyaWFsOyI+Jmd0OzoN
Cjwvc3Bhbj48L3NwYW4+PC9pPjwvZGl2Pg0KPGRpdj48YnIvPjwvZGl2Pg0KPGJsb2NrcXVvdGUg
c3R5bGU9ImJvcmRlci1sZWZ0OjFweCBzb2xpZCAjY2NjY2NjO21hcmdpbjowcHggMHB4IDBweCAw
LjhleDtwYWRkaW5nLWxlZnQ6MWV4OyI+DQo8ZGl2IHN0eWxlPSJkaXNwbGF5OmJsb2NrOyI+DQo8
ZGl2PjwvZGl2Pg0KPGRpdj48Yj5AU29raWxza2lsbDwvYj4gYXBwcm92ZWQgdGhpcyBwdWxsIHJl
cXVlc3QuPC9kaXY+DQo8ZGl2PtCy0LjQvdC10YHQuCDQsiDQvtC60YDQtdC80LjQuSDQutC+0LzQ
v9C+0L3QtdC90YIg0LXQu9C10LzQtdC90YLQuDwvZGl2Pg0KPGhyLz48ZGl2PkluIDxhIGhyZWY9
Imh0dHBzOi8vZ2l0aHViLmNvbS9Tb2tpbHNraWxsL3Byb2plY3QtZGFpbHktZW5lcmd5LWRvc2Uv
cHVsbC8xMSNkaXNjdXNzaW9uX3IxNDUyMDAxMTk0IiB0YXJnZXQ9Il9ibGFuayIgcmVsPSJub3Jl
ZmVycmVyIG5vb3BlbmVyIj5zcmMvcGFnZXMvU2lnbkluUGFnZS9TaWduSW5QYWdlLmpzeDwvYT46
PC9kaXY+DQo8cHJlIHN0eWxlPSJtYXJnaW46NXB4IDA7Ij4mZ3Q7ICsgICAgICAgICAgICAgICAg
ICBwbGF0Zm9ybToNCisgICAgICAgICAgICAgICAgJmx0Oy9wJmd0Ow0KKyAgICAgICAgICAgICAg
Jmx0Oy9kaXYmZ3Q7DQorICAgICAgICAgICAgICAmbHQ7U2lnbkluRm9ybSBvblN1Ym1pdD17aGFu
ZGxlRm9ybURhdGF9IC8mZ3Q7DQorICAgICAgICAgICAgICAmbHQ7cCBjbGFzc05hbWU9e2Nzcy5z
aWdudXBfb3B0aW9ufSZndDsNCisgICAgICAgICAgICAgICAgRG9u4oCZdCBoYXZlIGFuIGFjY291
bnQ/DQorICAgICAgICAgICAgICAgICZsdDtzcGFuJmd0Ow0KKyAgICAgICAgICAgICAgICAgICZs
dDtMaW5rIGNsYXNzTmFtZT17Y3NzLnNpZ251cF9saW5rfSB0bz0iL3NpZ251cCImZ3Q7DQorICAg
ICAgICAgICAgICAgICAgICBTaWduIFVwDQorICAgICAgICAgICAgICAgICAgJmx0Oy9MaW5rJmd0
Ow0KKyAgICAgICAgICAgICAgICAmbHQ7L3NwYW4mZ3Q7DQorICAgICAgICAgICAgICAmbHQ7L3Am
Z3Q7DQorICAgICAgICAgICAgJmx0Oy9kaXYmZ3Q7DQorICAgICAgICAgICZsdDsvZGl2Jmd0Ow0K
Kw0KKyAgICAgICAgICAmbHQ7ZGl2IGNsYXNzTmFtZT17Y3NzLndlbGNvbWVfYmdkfSZndDsNCjwv
cHJlPg0KPGRpdj7RhtC1INCy0LjQvdC10YHRgtC4INCy0LrQvtC80L/QvtC90LXQvdGCPC9kaXY+
DQo8aHIvPjxkaXY+SW4gPGEgaHJlZj0iaHR0cHM6Ly9naXRodWIuY29tL1Nva2lsc2tpbGwvcHJv
amVjdC1kYWlseS1lbmVyZ3ktZG9zZS9wdWxsLzExI2Rpc2N1c3Npb25fcjE0NTIwMDE2NTEiIHRh
cmdldD0iX2JsYW5rIiByZWw9Im5vcmVmZXJyZXIgbm9vcGVuZXIiPnNyYy9wYWdlcy9XZWxjb21l
UGFnZS9XZWxjb21lUGFnZS5qc3g8L2E+OjwvZGl2Pg0KPHByZSBzdHlsZT0ibWFyZ2luOjVweCAw
OyI+Jmd0OyAgICAgICAgICAgICAgJmx0Oy9kaXYmZ3Q7DQotICAgICAgICAgICAgICAgICAgICAm
bHQ7ZGl2Jmd0Ow0KLSAgICAgICAgICAgICAgICAgICAgJmx0O3AgY2xhc3NOYW1lPXtjc3Mudmll
d3N9Jmd0OzM1MCArJmx0Oy9wJmd0Ow0KLSAgICAgICAgICAgICAgICAgICAgJmx0O3AgY2xhc3NO
YW1lPXtjc3Mudmlld3NfdGV4dH0mZ3Q7VmlkZW8gdHV0b3JpYWwmbHQ7L3AmZ3Q7Jmx0Oy9kaXYm
Z3Q7ICAgIA0KLSAgICAgICAgICAgICAgICAgICAgJmx0Oy9kaXYmZ3Q7ICAgICAgDQotICAgICAg
ICAgICZsdDtkaXYgY2xhc3NOYW1lPXtjc3MuY2FsX2JveH0mZ3Q7DQotICAgICAgICAgICAgJmx0
O2RpdiBjbGFzc05hbWU9e2Nzcy5jYWxfZWxsaXBzfSZndDsNCi0gICAgICAgICAgICAmbHQ7c3Zn
IGNsYXNzTmFtZT17Y3NzLmljb25fcnVufSZndDsNCi0gICAgICAgICAgICAgICAgICZsdDt1c2Ug
Y2xhc3NOYW1lPXtjc3MuaWNvbl9ydW59IGhyZWY9e2Ake3Nwcml0ZX0jaWNvbi1ydW5uaW5nLXN0
aWNrLWZpZ3VyZS1zdmdyZXBvLWNvbS0xLTFgfSAvJmd0Ow0KLSAgICAgICAgICAgICAgICZsdDsv
c3ZnJmd0Ow0KKyAgICAgICAgICAmbHQ7L2RpdiZndDsNCisgICAgICAgICAgJmx0O2RpdiBjbGFz
c05hbWU9e2Nzcy53ZWxjb21lX2JnZH0mZ3Q7DQo8L3ByZT4NCjxkaXY+0LLQuNC90LXRgdGC0Lgg
0LIg0LrQvtC80L/QvtC90LXQvdGCPC9kaXY+DQo8ZGl2PjxzcGFuIHN0eWxlPSJmb250LXNpemU6
c21hbGw7Y29sb3I6IzY2NjsiPuKAlDxici8+UmVwbHkgdG8gdGhpcyBlbWFpbCBkaXJlY3RseSwg
PGEgaHJlZj0iaHR0cHM6Ly9naXRodWIuY29tL1Nva2lsc2tpbGwvcHJvamVjdC1kYWlseS1lbmVy
Z3ktZG9zZS9wdWxsLzExI3B1bGxyZXF1ZXN0cmV2aWV3LTE4MjEwOTEzNjIiIHRhcmdldD0iX2Js
YW5rIiByZWw9Im5vcmVmZXJyZXIgbm9vcGVuZXIiPnZpZXcgaXQgb24gR2l0SHViPC9hPiwgb3Ig
PGEgaHJlZj0iaHR0cHM6Ly9naXRodWIuY29tL25vdGlmaWNhdGlvbnMvdW5zdWJzY3JpYmUtYXV0
aC9BTzM1VUs3TzdNSTY3UjU2UkhaQVNDRFlPVEszUkFWQ05GU002QUFBQUFCQjJLQTJUT1ZISTJE
U01WUVdJWDNMTVY0M1lVRFZOUldGRVpMUk9WU1hHNUNTTVYzR1NaTFhITVlUUU1SUkdBNFRDTVpX
R0kiIHRhcmdldD0iX2JsYW5rIiByZWw9Im5vcmVmZXJyZXIgbm9vcGVuZXIiPnVuc3Vic2NyaWJl
PC9hPi48YnIvPllvdSBhcmUgcmVjZWl2aW5nIHRoaXMgYmVjYXVzZSB5b3UgYXV0aG9yZWQgdGhl
IHRocmVhZC48aW1nIHNyYz0iaHR0cHM6Ly9naXRodWIuY29tL25vdGlmaWNhdGlvbnMvYmVhY29u
L0FPMzVVSzJURkxKRDZITU40V0pFR09UWU9USzNSQTVDTkZTTTZBQUFBQUJCMktBMlRPV0dHMzNO
TlZTVzQ1QzdPUjRYQVpOUktCMldZM0NTTVZZWEtaTFRPUkpHSzVUSk1WMzJVWTNQTlZXV0szVFVM
NVVXSlRUTVJPVENFLmdpZiIgaGVpZ2h0PSIxIiB3aWR0aD0iMSIgYWx0PSIiLz48L3NwYW4+PHNw
YW4gc3R5bGU9ImZvbnQtc2l6ZTowO2NvbG9yOnRyYW5zcGFyZW50O2Rpc3BsYXk6bm9uZTtvdmVy
ZmxvdzpoaWRkZW47d2lkdGg6MDtoZWlnaHQ6MDttYXgtd2lkdGg6MDttYXgtaGVpZ2h0OjA7Ij5N
ZXNzYWdlIElEOiAmbHQ7U29raWxza2lsbC9wcm9qZWN0LWRhaWx5LWVuZXJneS1kb3NlL3B1bGwv
MTEvcmV2aWV3LzE4MjEwOTEzNjJAZ2l0aHViLmNvbSZndDs8L3NwYW4+PC9kaXY+DQo8L2Rpdj4N
CjwvYmxvY2txdW90ZT48L3NwYW4+PC9ib2R5PjwvaHRtbD4NCg==

--=-wVOo+DNbh/Yi1KH2DSm0--

**Pull request merged**